docs/resources/synthetics_private_location.md

```diff
@@ -30,13 +30,21 @@ resource "datadog_synthetics_private_location" "private_location" {
 ### Optional
 
 - **description** (String) Description of the private location.
+- **metadata** (Block List, Max: 1) The private location metadata (see [below for nested schema](#nestedblock--metadata))
```

Comment:
I don't think we'll be able to change this wording because it's auto-generated.  See [synthetics_test.md](https://github.com/DataDog/terraform-provider-datadog/blob/f89268e55913488cb66812b80d4a514a63472d9d/docs/resources/synthetics_test.md?plain=1) for many examples of this exact phrasing.